### ISSUE TITLE
post: TypeError: Cannot read property 'preloader' of undefined

### ISSUE THREAD

**Issue by davious** [rc0x03/node-osmosis#10]
I'm having trouble attempting to use the `post` method:

When I do
```js
osmosis.post('www.craigslist.org/about/sites');
```
I get
```
node-osmosis/lib/promise.js:17
if (typeof cb.preloader === 'function')
             ^
TypeError: Cannot read property 'preloader' of undefined
    at null.post (node-osmosis/lib/promise.js:17:16)
```

**Comment by bencrenshaw0313** [comment]
'www.craigslist.org/about/sites'

**Comment by rc0x03** [comment]
This should be fixed in version 0.0.5. Please update and try again.

**Comment by davious** [comment]
works in 0.0.5. Thanks
